### PR TITLE
Move away from `Luxor` for visualization

### LINKED ISSUE #11: Load graph and drawing packages lazily using Package Extensions
Package Extensions are introduced in yet unreleased Julia 1.9

### PR DESCRIPTION
**Pull request by mofeing**
This PR closes #11 (partially),#21 and #30 by moving away from `Luxor` (and the extra dependencies `MathTeXEngine` and `LaTeXStrings`) and using `Cobweb` for directly writing SVG code.

I've found out that `Luxor` is quite a heavy package. This PR substantially decreases package load and precompilation times to almost none, and eases dependency complexity.

`Cobweb` doesn't rely on `libxml2` for writing XML-like code (e.g. `EzXML`) so the package is veeery light.

### CODE REVIEW

File: src/Visualization.jl [hunk 1]
```diff
@@ -1,28 +1,73 @@
...
+        stroke: black;
+        stroke-width: 2px;
+    }
+
+    .lane {}
+    .virtual {}
+
+    .block {
+        stroke: black;
+        fill: white;
```
Comment (jofrevalles) [inline src/Visualization.jl:31]: Maybe make the background of the blocks `"transparent"` so it has the same background as the other lanes and wires?
```suggestion
        fill: transparent;
```

Comment (jofrevalles) [inline src/Visualization.jl:31]: Maybe make the background of the blocks `"transparent"` so it has the same background as the other lanes and wires?
```suggestion
        fill: transparent;
```

**Review by jofrevalles — CHANGES_REQUESTED** [review]

File: src/Visualization.jl [hunk 2]
```diff
@@ -1,28 +1,73 @@
...
-texname(::Type{Hz}) = L"H_Z"
+texname(::Type{Hz}) = """H<tspan font-size="60%" baseline-shift="sub">Z</tspan>"""
 
-texname(::Type{FSim}) = L"F_S"
+texname(::Type{FSim}) = """F<tspan font-size="60%" baseline-shift="sub">S</tspan>"""
 
 function draw end
 export draw
 
+const DRAWING_STYLE = h.style(
```
[the remark this answers is not in the document]
Comment (mofeing) [reply src/Visualization.jl:19]: I think is better to leave this `const` but to call it `DEFAULT_STYLE, and we add some functionality for customization.

File: src/Visualization.jl (hunk 1 above)
Comment (mofeing) [reply src/Visualization.jl:31]: For this PR, I think is a good idea but in the future I want to be able to reduce the number of elements in the drawing by making the lanes full-length lines (this also should partially solve #19).

**Review by jofrevalles — APPROVED** [review]

**Pull request merged**